go.py

```diff
@@ -0,0 +1,84 @@
+'''
+We need to design the board to be based on player skill level. Beginners = 9x9, Intermediate = 13x13, and Advanced = 19x19
```

Comment:
Nice I like this idea. We should probably also put this information in the readme

Reply:
I want to have a section for game rules in the readme